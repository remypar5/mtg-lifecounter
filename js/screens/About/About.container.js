import React from 'react';
import { StyleSheet } from 'react-native';
import HeaderButtons from 'react-navigation-header-buttons';
import FeatherIcon from 'react-native-vector-icons/Feather';

import About from './About';
import { COLOR_FOREGROUND } from '../../utils/constants';

const styles = StyleSheet.create({
    header: {
        backgroundColor: '#101011',
    },
    headerTitle: {
        color: COLOR_FOREGROUND,
    },
});

// Ignore the next line because react-router requires containers to be actual React Components
// eslint-disable-next-line react/prefer-stateless-function
export default class GameSettingsContainer extends React.Component {
    render() {
        return (
            <About />
        );
    }
}

GameSettingsContainer.navigationOptions = ({ navigation }) => ({
    title: 'Info',
    headerTitle: 'Info',
    headerLeft: (
        <HeaderButtons IconComponent={FeatherIcon} iconSize={28} color={COLOR_FOREGROUND}>
            <HeaderButtons.Item
                title="Back"
                iconName="arrow-left"
                onPress={() => navigation.goBack()}
            />
        </HeaderButtons>
    ),
    headerStyle: styles.header,
    headerTitleStyle: styles.headerTitle,
});
