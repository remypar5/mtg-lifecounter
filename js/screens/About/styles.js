import { StyleSheet } from 'react-native';

import { normalizeFontSize } from '../../utils';
import { FONT_SIZE_TINY, FONT_SIZE_SMALL } from '../../utils/constants';

export default StyleSheet.create({
    container: {
        paddingHorizontal: normalizeFontSize(FONT_SIZE_TINY),
    },
    heading: {
        fontSize: normalizeFontSize(FONT_SIZE_SMALL),
        marginBottom: normalizeFontSize(FONT_SIZE_TINY),
    },
    paragraph: {
        fontSize: normalizeFontSize(FONT_SIZE_TINY),
        marginBottom: normalizeFontSize(FONT_SIZE_TINY),
    },
    link: {
        fontSize: normalizeFontSize(FONT_SIZE_TINY),
        textDecorationLine: 'underline',
        color: 'red',
    },
});
