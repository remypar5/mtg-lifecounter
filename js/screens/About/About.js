import React from 'react';
import { ScrollView } from 'react-native';

import styles from './styles';
import bgiconSrc from '../Game/bgicon.png';
import { Text, PageContainer, BackgroundImage } from '../../components';

const { heading, paragraph, link } = styles;

const About = () => (
    <PageContainer>
        <BackgroundImage source={bgiconSrc} />
        <ScrollView>
            <Text type="heading" style={heading}>About</Text>
            <Text type="paragraph" style={paragraph}>
                The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of
                up to 6 players{"'"} life totals, and more will be added later.
            </Text>
            <Text type="paragraph" style={paragraph}>
                This project is for me to learn some new technologies, so if you have suggestions
                on how to improve the functionalities, please
                <Text type="paragraph" style={link}>contact me</Text>.
                I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate
                and review the app in the Play Store as honest as possible.
            </Text>

            <Text type="heading" style={heading}>Translations</Text>
            <Text type="paragraph" style={paragraph}>
                In the near future, I want to start supporting every language in which Magic: The
                Gathering cards are being printed. And Dutch. Because that{"'"}s my mother tongue.
                For this, I am looking for users who want to help me translate the few bits of
                text in this app for their own language. I{"'"}m still looking for users that can
                read/write Simplified Chinese, Traditional Chinese, French, German, Italian,
                Japanese, Korean, Portuguese, Russian and Spanish.
            </Text>

            <Text type="heading" style={heading}>Special thanks</Text>
            <Text type="paragraph" style={paragraph}>
                A special thanks goes to Rory, my colleague, who reviews my code every now and
                then. And to my friend Thijs, who designed everything in and around the MTG
                Lifecounter app.
            </Text>
        </ScrollView>
    </PageContainer>
);

export default About;
