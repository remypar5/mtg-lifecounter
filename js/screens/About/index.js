import About from './About';
import AboutScreen from './About.container';
import styles from './styles';

export {
    About,
    AboutScreen,
    styles,
};
