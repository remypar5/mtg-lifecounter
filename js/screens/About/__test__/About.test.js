import React from 'react';
import { shallow } from 'enzyme';

import About, { ContactMe } from '../About';

describe('<About />', () => {
    it('renders without crashing', () => {
        const component = shallow(<About />);
        expect(component).toMatchSnapshot();
    });
});

describe('<ContactMe />', () => {
    it('renders without crashing', () => {
        const component = shallow(<ContactMe />);
        expect(component).toMatchSnapshot();
    });
});
