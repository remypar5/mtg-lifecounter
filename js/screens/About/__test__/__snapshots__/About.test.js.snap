// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <About />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <BackgroundImage
          source={
            Object {
              "testUri": "../../../js/screens/Game/bgicon.png",
            }
          }
          style={null}
        />,
        <ScrollView>
          <Text
            style={
              Object {
                "fontSize": 135,
                "marginBottom": 90,
              }
            }
            type="heading"
          >
            About
          </Text>
          <Text
            style={
              Object {
                "fontSize": 90,
                "marginBottom": 90,
              }
            }
            type="paragraph"
          >
            The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players
            '
             life totals, and more will be added later.
          </Text>
          <Text
            style={
              Object {
                "fontSize": 90,
                "marginBottom": 90,
              }
            }
            type="paragraph"
          >
            This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please 
            <ContactMe />
            . I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.
          </Text>
          <Text
            style={
              Object {
                "fontSize": 135,
                "marginBottom": 90,
              }
            }
            type="heading"
          >
            Translations
          </Text>
          <Text
            style={
              Object {
                "fontSize": 90,
                "marginBottom": 90,
              }
            }
            type="paragraph"
          >
            In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that
            '
            s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I
            '
            m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.
          </Text>
          <Text
            style={
              Object {
                "fontSize": 135,
                "marginBottom": 90,
              }
            }
            type="heading"
          >
            Special thanks
          </Text>
          <Text
            style={
              Object {
                "fontSize": 90,
                "marginBottom": 90,
              }
            }
            type="paragraph"
          >
            A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.
          </Text>
        </ScrollView>,
      ],
      "style": Object {
        "paddingHorizontal": 90,
      },
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "source": Object {
            "testUri": "../../../js/screens/Game/bgicon.png",
          },
          "style": null,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": Array [
            <Text
              style={
                Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                }
              }
              type="heading"
            >
              About
            </Text>,
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players
              '
               life totals, and more will be added later.
            </Text>,
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please 
              <ContactMe />
              . I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.
            </Text>,
            <Text
              style={
                Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                }
              }
              type="heading"
            >
              Translations
            </Text>,
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that
              '
              s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I
              '
              m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.
            </Text>,
            <Text
              style={
                Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                }
              }
              type="heading"
            >
              Special thanks
            </Text>,
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.
            </Text>,
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": "About",
              "style": Object {
                "fontSize": 135,
                "marginBottom": 90,
              },
              "type": "heading",
            },
            "ref": null,
            "rendered": "About",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                "The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players",
                "'",
                " life totals, and more will be added later.",
              ],
              "style": Object {
                "fontSize": 90,
                "marginBottom": 90,
              },
              "type": "paragraph",
            },
            "ref": null,
            "rendered": Array [
              "The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players",
              "'",
              " life totals, and more will be added later.",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                "This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please ",
                <ContactMe />,
                ". I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.",
              ],
              "style": Object {
                "fontSize": 90,
                "marginBottom": 90,
              },
              "type": "paragraph",
            },
            "ref": null,
            "rendered": Array [
              "This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please ",
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {},
                "ref": null,
                "rendered": null,
                "type": [Function],
              },
              ". I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": "Translations",
              "style": Object {
                "fontSize": 135,
                "marginBottom": 90,
              },
              "type": "heading",
            },
            "ref": null,
            "rendered": "Translations",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": Array [
                "In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that",
                "'",
                "s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I",
                "'",
                "m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.",
              ],
              "style": Object {
                "fontSize": 90,
                "marginBottom": 90,
              },
              "type": "paragraph",
            },
            "ref": null,
            "rendered": Array [
              "In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that",
              "'",
              "s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I",
              "'",
              "m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.",
            ],
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": "Special thanks",
              "style": Object {
                "fontSize": 135,
                "marginBottom": 90,
              },
              "type": "heading",
            },
            "ref": null,
            "rendered": "Special thanks",
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "function",
            "props": Object {
              "children": "A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.",
              "style": Object {
                "fontSize": 90,
                "marginBottom": 90,
              },
              "type": "paragraph",
            },
            "ref": null,
            "rendered": "A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.",
            "type": [Function],
          },
        ],
        "type": [MockFunction],
      },
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <BackgroundImage
            source={
              Object {
                "testUri": "../../../js/screens/Game/bgicon.png",
              }
            }
            style={null}
          />,
          <ScrollView>
            <Text
              style={
                Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                }
              }
              type="heading"
            >
              About
            </Text>
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players
              '
               life totals, and more will be added later.
            </Text>
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please 
              <ContactMe />
              . I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.
            </Text>
            <Text
              style={
                Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                }
              }
              type="heading"
            >
              Translations
            </Text>
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that
              '
              s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I
              '
              m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.
            </Text>
            <Text
              style={
                Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                }
              }
              type="heading"
            >
              Special thanks
            </Text>
            <Text
              style={
                Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                }
              }
              type="paragraph"
            >
              A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.
            </Text>
          </ScrollView>,
        ],
        "style": Object {
          "paddingHorizontal": 90,
        },
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "source": Object {
              "testUri": "../../../js/screens/Game/bgicon.png",
            },
            "style": null,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": Array [
              <Text
                style={
                  Object {
                    "fontSize": 135,
                    "marginBottom": 90,
                  }
                }
                type="heading"
              >
                About
              </Text>,
              <Text
                style={
                  Object {
                    "fontSize": 90,
                    "marginBottom": 90,
                  }
                }
                type="paragraph"
              >
                The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players
                '
                 life totals, and more will be added later.
              </Text>,
              <Text
                style={
                  Object {
                    "fontSize": 90,
                    "marginBottom": 90,
                  }
                }
                type="paragraph"
              >
                This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please 
                <ContactMe />
                . I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.
              </Text>,
              <Text
                style={
                  Object {
                    "fontSize": 135,
                    "marginBottom": 90,
                  }
                }
                type="heading"
              >
                Translations
              </Text>,
              <Text
                style={
                  Object {
                    "fontSize": 90,
                    "marginBottom": 90,
                  }
                }
                type="paragraph"
              >
                In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that
                '
                s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I
                '
                m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.
              </Text>,
              <Text
                style={
                  Object {
                    "fontSize": 135,
                    "marginBottom": 90,
                  }
                }
                type="heading"
              >
                Special thanks
              </Text>,
              <Text
                style={
                  Object {
                    "fontSize": 90,
                    "marginBottom": 90,
                  }
                }
                type="paragraph"
              >
                A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.
              </Text>,
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "About",
                "style": Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                },
                "type": "heading",
              },
              "ref": null,
              "rendered": "About",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  "The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players",
                  "'",
                  " life totals, and more will be added later.",
                ],
                "style": Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                },
                "type": "paragraph",
              },
              "ref": null,
              "rendered": Array [
                "The MTG Lifecounter app is for Magic: The Gathering players. It helps keep track of up to 6 players",
                "'",
                " life totals, and more will be added later.",
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  "This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please ",
                  <ContactMe />,
                  ". I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.",
                ],
                "style": Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                },
                "type": "paragraph",
              },
              "ref": null,
              "rendered": Array [
                "This project is for me to learn some new technologies, so if you have suggestions on how to improve the functionalities, please ",
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "function",
                  "props": Object {},
                  "ref": null,
                  "rendered": null,
                  "type": [Function],
                },
                ". I will see if your idea fits my vision of the MTG Lifecounter. Even better: rate and review the app in the Play Store as honest as possible.",
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "Translations",
                "style": Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                },
                "type": "heading",
              },
              "ref": null,
              "rendered": "Translations",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": Array [
                  "In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that",
                  "'",
                  "s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I",
                  "'",
                  "m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.",
                ],
                "style": Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                },
                "type": "paragraph",
              },
              "ref": null,
              "rendered": Array [
                "In the near future, I want to start supporting every language in which Magic: The Gathering cards are being printed. And Dutch. Because that",
                "'",
                "s my mother tongue. For this, I am looking for users who want to help me translate the few bits of text in this app for their own language. I",
                "'",
                "m still looking for users that can read/write Simplified Chinese, Traditional Chinese, French, German, Italian, Japanese, Korean, Portuguese, Russian and Spanish.",
              ],
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "Special thanks",
                "style": Object {
                  "fontSize": 135,
                  "marginBottom": 90,
                },
                "type": "heading",
              },
              "ref": null,
              "rendered": "Special thanks",
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.",
                "style": Object {
                  "fontSize": 90,
                  "marginBottom": 90,
                },
                "type": "paragraph",
              },
              "ref": null,
              "rendered": "A special thanks goes to my colleague Rory, who reviews my code every now and then. And to my friend Thijs, who designed everything in and around the MTG Lifecounter app.",
              "type": [Function],
            },
          ],
          "type": [MockFunction],
        },
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`<ContactMe /> renders without crashing 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <ContactMe />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": "contact me",
      "href": "mailto:dev@parzinski.nl?subject=Suggestion%20to%20improve%20the%20MTG%20Lifecounter%20app",
      "style": null,
    },
    "ref": null,
    "rendered": "contact me",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "contact me",
        "href": "mailto:dev@parzinski.nl?subject=Suggestion%20to%20improve%20the%20MTG%20Lifecounter%20app",
        "style": null,
      },
      "ref": null,
      "rendered": "contact me",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
